packages/cyberstorm/src/components/componentStyles/Tag.module.css

```diff
@@ -1,62 +1,61 @@
 /* Common Tag styles */
-.tag {
+.root {
   display: flex;
   flex: none;
   flex-direction: row;
+
+  gap: var(--gap-2xs);
   align-items: center;
   justify-content: center;
-
   border: 2px solid;
   border-color: var(--border-color);
-  border-radius: var(--size-l);
+  border-radius: 999em;
   color: var(--text-color);
+
+  font: var(--font-body);
+  font-weight: var(--font-weight-bold);
   background-color: var(--bg-color);
 
   --text-color: var(--color-text--default);
   --bg-color: var(--color-surface--2);
   --border-color: transparent;
+
+  cursor: default;
 }
 
-.tag:hover {
+.root:hover {
   --bg-color: var(--color-surface--0);
   --border-color: var(--color-border-3);
 }
 
 /* Different Tag sizes */
 
 .small {
-  gap: var(--size-tiny);
-  padding: var(--size-tiny) var(--text-xxs);
-  font: normal 700 var(--text-xxs) Inter, sans-serif;
-  line-height: var(--size-tiny);
+  padding: var(--padding-tiny) var(--padding-2xs);
```

Comment:
Should there be `tiny` and `2xs` styled naming mixed anymore?

Reply:
the idea was to have "tiny" as a shortcut, easier to remember than "5xs". 🤷‍♂️ "tiny" would be used as a "default" smallest size.